src/libraries/System.Private.Runtime.InteropServices.JavaScript/src/System.Private.Runtime.InteropServices.JavaScript.csproj

```diff
@@ -5,7 +5,7 @@
     <TargetFrameworks>$(NetCoreAppCurrent)-Browser</TargetFrameworks>
     <NoWarn>$(NoWarn);CA1419</NoWarn> <!-- // TODO https://github.com/dotnet/roslyn-analyzers/issues/5232: not intended for use with P/Invoke -->
   </PropertyGroup>
-  <ItemGroup Condition="'$(TargetsBrowser)' == 'true'">
+  <ItemGroup>
```

Comment:
This assembly is browser only. So I wonder if this condition made any difference even before. 
Note that the classes here should not become public in current shape.

Reply:
Right, those condition didn't do anything, that's why I'm removing them. Initially I added a rid-agnostic tfm to this library as well but as it doesn't have a ref assembly I can't make it use the PNSE throwing assembly feature. Hence I reverted my changes and just left this clean-up be.

Reply:
got it. We need to make the `JSObject` etc public API worthy first, before we could make it ref assembly and drop `.Private` from it's name.